### PR TITLE
Create start-posting-notes.md

### PR DESCRIPTION
**Pull request by boston-wine**
Re-do on creating the "Start Posting Notes" page. Closed the old PR, synced to your current repo.

I think this is right, now, as the only difference between my fork and your original is this page.

Let me know if I should update the sidebar like you had [mentioned](https://github.com/erskingardner/nostr-how/pull/174#issuecomment-1928950394) on the old PR. If so, I found [where](https://github.com/boston-wine/nostr-how/blob/main/src/lib/components/Sidebar.svelte) to add the page to the sidebar, but was not sure how I should edit the line that identifies the page. Each menu item is formatted like this:

`{ href: "/guides/setup-zapping-wallet", localeString: "nav.guideSetupWallet" },
`
So presumably, mine should look something like this, but I'm not clear on what the text at the end should actually say, or if I need to define it elsewhere (like on the actual Start Posting Notes page).

`{ href: "/guides/start-posting-notes", localeString: "nav.guide__?_?_?__" },`

Anyway, if it makes no difference to you, @erskingardner, it's probably simpler for me to learn/cement the PR process by just working on this page by itself, and then making a separate, second change to update the sidebar nav, so that I get the practice with syncing, forking, editing, and opening pull requests.

Oh, and let me know if you'd advise any changes to the copy itself 🫡

### CODE REVIEW

File: src/routes/pages/en/guides/start-posting-notes.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+At the time of this writing, the hashtag #grownostr is a popular catch-all, used for a wide range of topics. (Since nostr was built by bitcoiners, there is still a lot of bitcoin-related discussion, so we use #grownostr to identify various other topics). 
+
+A few other popular hashtags include #nature, #artstr, #photography, #foodstr, #dogstr, and #AiArt, for example. Search or post with #introductions and #plebchain to find new friends.
+
+Ultimaetly, you can Use any hashtags that help you find interesting content. If you don't see much, try adding "-str" to the end. For example, if you're looking for cute cat photos, search for #catstr :cat2:
+
+
+## [§](#tagging-other-users) Tagging Other Users
+
+Tagging other users is still a work in progress (as of this writing in early 2024). On most clients, if you're already following someone, then using @ followed by their display name will work. Of course, if you use @ followed by their npub, that's a surefire way to tag your friend (but of course that assumes you have already saved their npub to your clipboard).
```
Comment (erskingardner) [inline src/routes/pages/en/guides/start-posting-notes.md:34]: I think that it'd be good to mention briefly that mentions can also be done by using the `nostr:note1...` or nostr:npub...` syntax. You don't need to go into detail on how to get that data but that way people will recognize it if they see it.

**Review by erskingardner — COMMENTED** [review]
One small comment but otherwise looks good to me.

**Comment by erskingardner** [comment]
> Let me know if I should update the sidebar like you had https://github.com/erskingardner/nostr-how/pull/174#issuecomment-1928950394 on the old PR. If so, I found [where](https://github.com/boston-wine/nostr-how/blob/main/src/lib/components/Sidebar.svelte) to add the page to the sidebar, but was not sure how I should edit the line that identifies the page. Each menu item is formatted like this:

Ok... unfortunately this is where this repo gets a bit complicated because of the translations. The localeString values are stored in the `/src/lib/locales/` directory. Each file in there contains the UI strings in the language that they correspond to. So you're right, you just need to add the URL and then create a new localestring value in the `nav` object of the `en.json` file.

For now, I think that it's ok that we don't create translations for this page. When there isn't a page in another language the page shows a message saying that it's not been translated yet, which is ok. (I should probably add a new call to action there to make it clear that we want translations). 

I'd say that you should go ahead and practice making another commit or two on this PR to change the thing I made a comment about and then add the link to the sidebar. Adding those two commits is as simple as `git commit -am 'message'` and `git push`. That'll update this PR.

**Comment by erskingardner** [comment]
Is this PR still on the go?

**Comment by erskingardner** [comment]
@boston-wine going to close this for now, feel free to reopen if you get back to it.

**Comment by boston-wine** [comment]
Hey Jeff, my B. Thought I answered this one earlier but totally fair to close for inactivity. When I can give it the attention deserved, I’ll reopen and submit anew 🫡

**Pull request closed without merging**